spec.html

```diff
@@ -31031,6 +31109,37 @@ <h1>lastIndex</h1>
         <p>The value of the `lastIndex` property specifies the String index at which to start the next match. It is coerced to an integer when used (see <emu-xref href="#sec-regexpbuiltinexec"></emu-xref>). This property shall have the attributes { [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
       </emu-clause>
     </emu-clause>
+
+    <emu-clause id="sec-regexp-identifier-names-static-semantics-stringvalue">
+      <h1>Static Semantics: StringValue</h1>
+      <emu-see-also-para op="StringValue"></emu-see-also-para>
+      <emu-grammar>
+        RegExpIdentifierName[U] ::
+          RegExpIdentifierStart[?U]
+          RegExpIdentifierName[?U] RegExpIdentifierPart[?U]
+      </emu-grammar>
+      <emu-alg>
+        1. Return the String value consisting of the sequence of code units corresponding to |RegExpIdentifierName|. In determining the sequence any occurrences of `\\` |RegExpUnicodeEscapeSequence| are first replaced with the code point represented by the |RegExpUnicodeEscapeSequence| and then the code points of the entire |RegExpIdentifierName| are converted to code units by UTF16Encoding each code point.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-backreference-matcher" aoid="BackreferenceMatcher">
+      <h1>Runtime Semantics: BackreferenceMatcher Abstract Operation</h1>
+      <p>The abstract operation BackreferenceMatcher takes one argument, an integer _n_, and performs the following steps:</p>
+      <emu-alg>
+          1. Return an internal Matcher closure that takes two arguments, a State _x_ and a Continuation _c_, and performs the following steps:
+            1. Let _cap_ be _x_'s _captures_ List.
+            1. Let _s_ be _cap_[_n_].
+            1. If _s_ is *undefined*, return _c_(_x_).
+            1. Let _e_ be _x_'s _endIndex_.
+            1. Let _len_ be the number of elements in _s_.
+            1. Let _f_ be _e_+_len_.
+            1. If _f_&gt;_InputLength_, return ~failure~.
+            1. If there exists an integer _i_ between 0 (inclusive) and _len_ (exclusive) such that Canonicalize(_s_[_i_]) is not the same character value as Canonicalize(_Input_[_e_+_i_]), return ~failure~.
```

Comment:
and the `e + i`?

Reply:
Done, but keep in mind that this is now inconsistent with other similar parts of the RegExp section. There is an open issue to fix this throughout the spec: https://github.com/tc39/ecma262/issues/925